Update Jython to latest 2.7.2

Update in our code here:  https://github.com/OpenRefine/OpenRefine/blob/3ca08f6ff189cc1bd806a1d49d2ead538882c0e2/extensions/jython/pom.xml#L124

New features and bugs fixed in Jython 2.7.2 since 2.7.1 are here:
https://github.com/jythontools/jython/blob/v2.7.2/NEWS

Agree, I don't think it will be a problem.... BUT, I cannot get our JythonEvaluable tests to even work without causing Maven errors.  Are our test classes working for the Jython extension on your side? 

Given:  `mvn verify -f "e:\GitHub Repos\OpenRefine\extensions\jython\pom.xml"`

Result: 
```
...
{snip}

[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running TestSuite
[TestNGClassFinder] Warning: Can't link and determine methods of class com.google.refine.jython.JythonEvaluable
[ERROR] Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 1.45 s <<< FAILURE! - in TestSuite
[ERROR] testWrappedObjectsHaveAttributes(com.google.refine.jython.JythonAttributeTest)  Time elapsed: 0.017 s  <<< FAILURE!
java.lang.Error:
Unresolved compilation problems:
        Row cannot be resolved to a type
        Row cannot be resolved to a type
        Cell cannot be resolved to a type
        Cell cannot be resolved to a type
        The constructor JythonHasFieldsWrapper(HasFields, Properties) refers to the missing type HasFields
        Evaluable cannot be resolved to a type

        at com.google.refine.jython.JythonAttributeTest.testWrappedObjectsHaveAttributes(JythonAttributeTest.java:34)
```

--- a/extensions/jython/pom.xml
+++ b/extensions/jython/pom.xml
@@ -121,7 +121,7 @@
     <dependency>
       <groupId>org.python</groupId>
       <artifactId>jython-standalone</artifactId>
-      <version>2.7.1</version>
+      <version>2.7.2</version>
     </dependency>
 
     <dependency>

